docs/syntax-reference.rst

```diff
@@ -243,13 +245,15 @@ The supported column data types are:
 -  ``BIGINT``
 -  ``DOUBLE``
 -  ``VARCHAR`` (or ``STRING``)
--  ``ARRAY<ArrayType>`` (JSON and AVRO only. Index starts from 0)
+-  ``ARRAY<ArrayType>`` (JSON and AVRO only. Index starts from 1)
```

Comment:
Why do we have this listed twice in the docs? Fancy pulling this out into its own section?